### PR TITLE
Fix #1683: Document configuration of OIDC activation

### LINKED ISSUE #1683: Document configuration of OIDC activation
A follow-up to #1642

### PR DESCRIPTION
**Pull request by banterCZ**

### CODE REVIEW

File: docs/OIDC-Activation.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+PowerAuth protocol allows activation using OpenID Connect (OIDC) protocol.
+
+
+## OIDC Activation Configuration
+
+To enable OIDC activation, an entry with the key `oauth2_providers` must exist in the table `pa_application_config`.
+Mind that this table supports encryption.
+
+- `providerId` - (Required) Identification of the configuration record, used as a key for the configuration, known by the mobile application.
```
Comment (romanstrobl) [inline docs/OIDC-Activation.md:11]: Can se separate the sections into required and optional parameters? It's a bit strange to use the (Required) or (Optional) word each time.

File: docs/OIDC-Activation.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+- `clientId` - (Required) The client identifier.
+- `clientSecret` - (Required) The client secret.
+- `issuerUri` - (Required) URI for the OpenID Connect 1.0 provider or the OAuth 2.0 Authorization Server; when `/.well-known/openid-configuration` endpoint exposed, other optional URI parameters are configured based on the response.
+- `redirectUri` - (Required) URI for the redirection endpoint.
+- `clientAuthenticationMethod` - (Optional) If empty, `client_secret_basic`.
```
Comment (romanstrobl) [inline docs/OIDC-Activation.md:16]: The sentence has no verb but it would improve the understanding.

**Review by romanstrobl — DISMISSED** [review]
Looks good with 2 suggestions.

File: docs/OIDC-Activation.md (hunk 1 above)
Comment (banterCZ) [reply docs/OIDC-Activation.md:11]: A good idea, I will improve that.

**Review by romanstrobl — APPROVED** [review]
Great!

**Pull request merged**